test/module/irohad/pending_txs_storage/pending_txs_storage_test.cpp

```diff
@@ -130,6 +129,94 @@ TEST_F(PendingTxsStorageFixture, InsertionTest) {
   }
 }
 
+/**
+ * PaginationMeta works in PendingTxsStorage
```

Comment:
I suggest more information in the line:
```suggestion
 * Timestamps in PaginationMeta works in PendingTxsStorage
```